<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect><x>0</x><y>0</y><width>600</width><height>400</height></rect>
        </property>
        <property name="windowTitle"><string>Music Player</string></property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QVBoxLayout" name="verticalLayout">

                <!-- Playlist -->
                <item><widget class="QListWidget" name="listWidget"/></item>

                <!-- Progress slider -->
                <item>
                    <widget class="QSlider" name="progressSlider">
                        <property name="orientation"><enum>Qt::Horizontal</enum></property>
                    </widget>
                </item>

                <!-- Buttons row -->
                <item>
                    <layout class="QHBoxLayout" name="buttonLayout">
                        <item><widget class="QPushButton" name="btnOpen"><property name="text"><string>Open</string></property></widget></item>
                        <item><widget class="QPushButton" name="btnPlay"><property name="text"><string>Play</string></property></widget></item>
                        <item><widget class="QPushButton" name="btnPause"><property name="text"><string>Pause</string></property></widget></item>
                        <item><widget class="QPushButton" name="btnStop"><property name="text"><string>Stop</string></property></widget></item>
                        <item><widget class="QPushButton" name="btnPrev"><property name="text"><string>⏮ Prev</string></property></widget></item>
                        <item><widget class="QPushButton" name="btnNext"><property name="text"><string>⏭ Next</string></property></widget></item>
                    </layout>
                </item>

                <!-- Volume slider -->
                <item>
                    <widget class="QSlider" name="volumeSlider">
                        <property name="minimum"><number>0</number></property>
                        <property name="maximum"><number>100</number></property>
                        <property name="value"><number>50</number></property>
                        <property name="orientation"><enum>Qt::Horizontal</enum></property>
                    </widget>
                </item>

            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar"/>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
